internal/services/loganalytics/log_analytics_linked_storage_account_resource.go

```diff
@@ -38,17 +39,18 @@ func resourceLogAnalyticsLinkedStorageAccount() *pluginsdk.Resource {
 
 		Schema: map[string]*pluginsdk.Schema{
 			"data_source_type": {
-				Type:     pluginsdk.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:             pluginsdk.TypeString,
+				Required:         true,
+				ForceNew:         true,
+				DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
how come this is required?

Reply:
@katbyte , it's very weird. When I removed "DiffSuppressFunc: suppress.CaseDifference,", all TCs passed on my local but some of TCs would fail in Teamcity. Is there any suggestion?